Robot2018/src/org/usfirst/frc/team199/Robot2018/commands/RunScript.java

```diff
@@ -55,7 +55,7 @@ public RunScript(String scriptName) {
 	    			case "end":
```

Comment:
Also, is the "wait" case option not merely repetitive of the delay option in Autonomous? That Wait command would be run before this RunScript one...

Reply:
the wait command simply adds more flexibility by allowing you to delay between commands instead of just in the start.